### PR TITLE
Make login page appear before main

### PR DESCRIPTION
**Pull request by ClarabelleCheng-Yue**
Login page now appears before entering main page

### CODE REVIEW

**Comment by JulianKnodt** [comment]
This looks good, except you should encrypt the password before actually storing it on the server

File: server/index.js [hunk 1]
```diff
@@ -37,32 +21,29 @@ var urlencodedParser = bp.urlencoded({ extended: false });
 app.use(urlencodedParser);
 
 //initialize session
-app.use(session({secret: 'secret'}));
-
-//serve static assets
-app.use(express.static('../Public'));
```
Comment (p-herbert) [inline server/index.js]: I think this line should be:

`
app.use(express.static('Public'));
`

File: server/index.js [hunk 2]
```diff
@@ -37,32 +21,29 @@ var urlencodedParser = bp.urlencoded({ extended: false });
...
-    next();  
-  } else {
-    res.redirect('/login');
-  }
-};
-
-
-//-------------------------- ROOT -------------------------//
-app.get('/', restrict, function(req, res) {
-  res.sendfile(path.resolve('../Public/index.html'));
```
Comment (p-herbert) [inline server/index.js]: Now we can serve static assets with:

`
res.sendfile('Public/index.html');
`

File: server/index.js [hunk 3]
```diff
@@ -166,15 +155,27 @@ app.post('/signup', function(req, res) {
   });
 });
 
+//-------------------------- ROOT -------------------------//
+app.get('/', function(req, res) {
+  if (!req.session.username && !req.session.pw) {
+    console.log('redirecting to log in');
+    res.redirect('/login');  
+  } else {
+    res.sendfile(path.resolve('../Public/index.html'));
```
Comment (p-herbert) [inline server/index.js]: `
 res.sendfile('Public/index.html'); 
`

**Comment by ClarabelleCheng-Yue** [comment]
Hi Pete,

I tried your suggestions, but then the server complains it can't find the files. 
screen shot 2016-09-08 at 10 17 36 pm

**Comment by p-herbert** [comment]
Hmm that is strange. It serves everything on my machine with those edits.

**Comment by oldchevy** [comment]
Yeah we ran into this problem. I think it's because the server is running in its own directory, not top level. So specifying 'Public' wouldn't work, there is no Public folder where the server is running

**Pull request closed without merging**